src/views/index.blade.php

```diff
@@ -128,14 +128,15 @@
                         <table id="decomposer" class="table table-hover table-bordered">
                             <thead>
                                 <tr>
-                                    <th>Package Name : Version</th>
+                                    <th>Package Name : Version : Version Installed</th>
                                     <th>Dependency Name : Version</th>
                                 </tr>
                             </thead>
                             <tbody>
+
                             @foreach($packages as $package)
                                 <tr>
-                                    <td>{{ $package['name'] }} : <span class="label ld-version-tag">{{ $package['version'] }}</span></td>
+                                    <td>{{ $package['name'] }} : <span class="label ld-version-tag">{{ $package['version'] }}</span> : <span class="label ld-version-tag">{{ $package['version-installed']->version }}</span> </td>
```

Comment:
I guess only `$package['version-installed']` would work. No need of `->version` again at the end. Not sure, make sure you test it :) Will be adding tests to this package really soon

Reply:
Hey mate, I'm actually taking the entire package object and passing it across, so you could effectively get more than the version: 
```
"aws/aws-sdk-php" => {#716 ▼
    +"name": "aws/aws-sdk-php"
    +"version": "3.40.0"
    +"source": {#721 ▶}
    +"dist": {#719 ▶}
    +"require": {#714 ▶}
    +"require-dev": {#724 ▶}
    +"suggest": {#720 ▶}
    +"type": "library"
    +"extra": {#725 ▶}
    +"autoload": {#727 ▶}
    +"notification-url": "https://packagist.org/downloads/"
    +"license": array:1 [▶]
    +"authors": array:1 [▶]
    +"description": "AWS SDK for PHP - Use Amazon Web Services in your PHP project"
    +"homepage": "http://aws.amazon.com/sdkforphp"
    +"keywords": array:8 [▶]
    +"time": "2017-11-27T09:20:45+00:00"
  }
```

I can pass just the version across if you prefer it that way, I normally lean to more info so it's easy to extend on.